### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by kiboma2021**
- Add a new JavaScript file and import it as a module:
       (it will contain methods related to the status updates (completed: true / false)).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @kiboma2021, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Good splitting of functions
- [ ] Features almost implemented

## Required Changes ♻️
- [ ] Your PR doesn't actually include all your changes for this project; Let's commit often when you complete a feature
- [ ]  The event listener on the checkbox should be the `change`, you are currently using the `click` event. Let's fix that
- [ ] The function for the "Clear all completed" should be using the `Array.filter()` method as it is in the requirements.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kiboma2021** [comment]
@EricMbouwe , i have used the filter method as per the screenshot below. Is this sufficient? Please advise as I am getting confused

image

image

**Comment by kiboma2021** [comment]
@EricMbouwe , please respond ASAP since I need to submit the project

**Comment by kiboma2021** [comment]
@EricMbouwe , I am resubmitting the project since your instructions on **filter** are not clear. Hope the next PR will give clear instructions on what needs to be done aside from what I have done. Thank you

File: src/status_update.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  const todos = document.querySelectorAll(".description");
+  for (let i=0; i<todos.length; i++) {
+    if(todos[i].classList.contains('check-to-do')) {
+      localData[i].complited = true;
+    } else {
+      localData[i].complited = false;
+    }
+  }
+  localStorage.setItem('List',JSON.stringify(localData));
+}
```
Comment (AmaduKamara) [inline src/status_update.js:13]: # Optional Changes
Good job working on the changes so far. 
- [ ] I noticed that when I marked a todo as completed, it gets marked and the actions also reflect on the localStorage. But when I refresh the page, the todos marked as completed are not fetched and displayed on the screen as it's in LocalStorage. It is good that when we make a todo as completed, it gets stored to localStorage and when the page refreshes, the same todo should be marked as completed. The previous reviewer did not spot that, please work on that. I trust you can. 👍

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @kiboma2021,

YesYouAreGIF
 
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlight
✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR description

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**